packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -399,12 +411,75 @@ class EditableText extends StatefulWidget {
   /// container will start with enough vertical space for one line and
   /// automatically grow to accomodate additional lines as they are entered.
   ///
-  /// If it is not null, the value must be greater than zero. If it is greater
-  /// than 1, it will take up enough horizontal space to accomodate that number
-  /// of lines.
+  /// If this is not null, the value must be greater than zero, and it will lock
+  /// the input to the given number of lines. Setting [minLines] as well allows
+  /// the input to grow between the indicated range.
+  ///
+  /// The full set of behaviors possible with [minLines] and [maxLines] are as
+  /// follows.
+  ///
+  /// Input that occupies a single line and scrolls horizontally as needed
+  /// ```dart
+  /// TextField()
```

Comment:
This is documentation for the EditableText class (too). Probably need some qualifying statement early on that explains that the examples apply equally well to EditableText or TextFormField.

Generally speaking I think we're talking about the "input widget's [EditableText]" rather than just "Input".